api/src/lib.rs

```diff
@@ -55,6 +55,7 @@ pub mod graph;
 pub mod ns;
 pub mod parser;
 pub mod prefix;
+#[doc(hidden)]
```

Comment:
Why hide it? Granted, it is not properly documented for the moment, but I think it is good to document that a prelude exists and what it contains (even if crudely).

As a reference, the standard prelude *is* included in the documentation; https://doc.rust-lang.org/std/prelude/index.html

Reply:
it was because the individual items in the prelude came up redundantly in the search. Perhaps it's better to hide them instead? I see your point regardless